### ISSUE TITLE
dev log #5 - language - feedback

### ISSUE THREAD

**Issue by underscorediscovery** [luxeengine/support#23]
Post your feedback and questions on [luxe dev log # 5 - language](https://luxeengine.com/dev-log-5/) here

**Comment by rickms** [comment]
Reading up on wren...  One thing that jumped out is the lack of AOT compiling. Having to ship source code with games is a bit underwhelming.  Anything in store to alleviate that concern?

**Comment by underscorediscovery** [comment]
It can actually compile directly to bytecode, and using that is a small step away right now but doable.

The parcels that assets are shipped as with 1.0 are not plain source files on disk. They're packed together, obfuscated (and optionally compressed etc). The plan is to also optionally store the scripts as bytecode as well (or that being the default in release builds, it depends).

**Comment by tanis2000** [comment]
Being able to transpile wren to C would also be a nice choice both for distribution and eventually performance gain. Just my 2 cents :)

**Comment by underscorediscovery** [comment]
yea I've tinkered with the idea already actually. There's also things like [sljit](https://github.com/mmcco/sljit), but they all have trade offs and platform specific concerns that push it to the wishlist immediate term (for me, others are free to explore that!). Since performance has gotten in my way yet at all it's something that'll come later too :)

**Comment by tanis2000** [comment]
Yes, I remember that sljit thing. It sounds like something worth checking out in the middle/long run.
